MailReporter/BusinessLogic/DiExtenstions.cs

```diff
@@ -11,6 +13,7 @@ public static IServiceCollection AddBusinessLogic(this IServiceCollection servic
         {
             return service
                 .AddScoped<IJobExecutionService, JobExecutionService>()
+                .AddScoped<ISendInBlueService,SendInBlueService>()
```

Comment:
There should be a space after the comma. You should even receive a warning about format problems during build.

Alternative 1: Before saving a file, you can press Ctrl (E + D)
Alternative 2: There's an extension called "Format document on Save", which presses ctrl E+D for you when you save.
Alternative 3: Create a new code cleanup profile with just "Format Document" fixer inside:
![image](https://user-images.githubusercontent.com/1922822/169480268-457c81bf-f5d1-482f-823d-f534a5f08791.png)
And then choose that on this option:
![image](https://user-images.githubusercontent.com/1922822/169480372-dbd2b6df-9557-4b83-88be-02c180fbe5fe.png)

Reply:
Fixed the formatting.